contracts/service_node_rewards.py

```diff
@@ -26,6 +26,14 @@ def __init__(self, provider_url: str, contract_address: str):
         self.contract         = self.web3.eth.contract(address=self.contract_address, abi=abi)
         self.address_map      = {}
 
+    def node_ids(self):
```

Comment:
For pattern matching the python implementation to the smart contract can we rename this `allServiceNodeIDs()`